File: resume-across-hosts.py
from claude_agent_sdk import list_sessions, rename_session
from datetime import datetime

sessions = list_sessions(directory=r"D:\github\far-force", limit=5)

for session in sessions:
    modified = datetime.fromtimestamp(session.last_modified / 1000).strftime("%Y-%m-%d %H:%M")
    print(f"{session.summary} - {modified} ({session.session_id})")

if sessions:
    rename_session(sessions[0].session_id, "Auth module refactor - JWT approach")
    print(f"Renamed session {sessions[0].session_id}")

File: resume-across-hosts.mjs
import { listSessions, renameSession } from "@anthropic-ai/claude-agent-sdk";

const sessions = await listSessions({ dir: "D:\\github\\far-force", limit: 5 });

for (const session of sessions) {
  const modified = new Date(session.lastModified).toLocaleString();
  console.log(`${session.summary} — ${modified} (${session.sessionId})`);
}

if (sessions.length > 0) {
    await renameSession(sessions[0].sessionId, "Auth module refactor – JWT approach", {
    dir: "D:\\github\\far-force"
    });
  console.log(`Renamed session ${sessions[0].sessionId}`);
}